tests/test_text_generation_example.py

```diff
@@ -71,6 +71,9 @@
         "torch_compile_distributed": [
             ("meta-llama/Llama-2-7b-hf", 39.72973199515235),
         ],
+        "distributed_tp": [
+            ("/mnt/weka/data/pytorch//llama2/Llama-2-7b-hf/", 1856.8140409694543),
```

Comment:
```suggestion
            ("meta-llama/Llama-2-7b-hf", 1856.8140409694543),
```
This is your local path, let's switch to the hub id.

Reply:
> This is your local path, let's switch to the hub id.

Done